scarb/build.rs

```diff
@@ -91,3 +92,20 @@ fn ident(id: &str) -> String {
     }
     ident
 }
+
+fn check_versions(comp_ver: &str, path: PathBuf) {
+    let core_toml: toml::Value =
+        toml::from_str(&fs::read_to_string(path).expect("Could not read Core library Scarb.toml"))
```

Comment:
```suggestion
        toml::from_str(&fs::read_to_string(path).expect("Could not read `core` Scarb.toml."))
```

nit: that's how we usually refer to this package in messages :)
nit2: We use "Full sentences with dot." for panics and "lower case without dot" for errors